src/test/java/SignUp.java

```diff
@@ -33,27 +39,23 @@ public static void SignUpWrongEmail() throws InterruptedException {
         WebDriver driver = new ChromeDriver();
         driver.get("https://deens.com/?internaltraffic");
         driver.manage().window().maximize();
+        Thread.sleep(1000);
 
-        WebElement signup = driver.findElement(By.cssSelector("[href*='register']"));
-        signup.click();
-        signup.sendKeys("AlmasMendy");
+        driver.findElement(By.cssSelector("[href*='register']")).click();;
         Thread.sleep(1000);
 
-        WebElement email = driver.findElement(By.cssSelector("[id=\'email\']"));
-        email.click();
-        email.sendKeys("gmail.com");
+        driver.findElement(By.cssSelector("[id=\'username\']")).sendKeys("Alm");
```

Comment:
you use the same locators in each test. It would be better to create static variable and store locator there and use it in different plases

Reply:
private static By userNameField = By.cssSelector("[id=\'username\']");